adodbapi/quick_reference.md

```diff
@@ -72,13 +70,13 @@ blocking installation of the redistributable provider.
 - To use any ODBC driver from 64 bit Python, you also need the MSDASQL
 provider. It is shipped with Server 2008, and Vista and later. For
 Server 2003, You will have to [download
-it](http://www.microsoft.com/en-us/download/details.aspx?id=20065) [from
-Microsoft.]{lang="en-US"}
+it](https://web.archive.org/web/20140205114906/http://www.microsoft.com:80/en-us/download/details.aspx?id=20065) from
+Microsoft.
```

Comment:
I couldn't find a modern equivalent, and the direct download link doesn't seem to have been saved on the Internet Archive (at least not from a first pass)
But I think removing concerns for the EOL Windows Server 2003 might be better overall

Reply:
Yeah, I think you can just replace from "It is shipped with Server 2008..." on with something like "you will need to find and install that provider."

Reply:
Linked removed by 

https://learn.microsoft.com/en-us/sql/connect/connect-history?view=sql-server-ver16
> MSDASQL ships with the Windows operating system, and Windows Server 2008 and Vista SP1 were the first Windows releases to include a 64-bit version of the technology.

I could possibly just remove the entire point. Or shorten it to
```md
- To use any ODBC driver from 64 bit Python, you also need the MSDASQL
provider, which is shipped with Windows.
```